lib/planner/write-plans-to-disk.js

```diff
@@ -7,7 +7,8 @@ module.exports = function writePlansToDisk(plans) {
     let date = Date.now();
     let fileCount = 1;
     for (let plan of plans) {
-        let filePath = path.resolve(process.env.PLANNER_OUTPUT_PATH, `${date}-test-case-${fileCount++}.json`);
+        let filePath = path.resolve(process.env.PLANNER_OUTPUT_PATH,
+            `${date}-simulato-${fileCount++}_${plans.length}.json`);
```

Comment:
Should this be `simulato-${date}`? Is that what we settled on?

Reply:
Discussed